service/internal/checksum/middleware_compute_input_checksum_test.go

```diff
@@ -514,8 +449,9 @@ func TestComputeInputPayloadChecksum(t *testing.T) {
 						return r
 					}(),
 				},
-				expectErr:      "unseekable stream is not supported",
-				expectBuildErr: true,
+				expectContentLength: -1,
```

Comment:
Why does unseekable stream works now when it didn't in the past?

Reply:
Old code would not calculate checksum for unseekable stream unless required by codegen. We should not return error when user send unseekable stream now